test/cli/repl/t10.3-flatten/expected

```diff
@@ -97,18 +97,28 @@ for_facts (conv$2@{Time}, conv$1@{FactIdentifier}, conv$0$simp$60@{Error}, conv$
     write acc$c$conv$11$simp$14 = flat$2$simp$40;
   }
   read acc$conv$26$simp$15 = acc$conv$26$simp$15 [Buf 3 FactIdentifier];
+  
   write acc$conv$26$simp$15 = Buf_push#@{Buf 3 FactIdentifier} acc$conv$26$simp$15 conv$1;
   read acc$conv$26$simp$16 = acc$conv$26$simp$16 [Buf 3 Error];
+  
+  
   write acc$conv$26$simp$16 = Buf_push#@{Buf 3 Error} acc$conv$26$simp$16 conv$0$simp$60;
   read acc$conv$26$simp$17 = acc$conv$26$simp$17 [Buf 3 Int];
+  
+  
   write acc$conv$26$simp$17 = Buf_push#@{Buf 3 Int} acc$conv$26$simp$17 conv$0$simp$61;
+  
+  
+  
+  
```

Comment:
yeah - I think it is because of weird nesting of blocks in the pretty printed code. maybe something like `Block [Block [], Block [], Block []]`?
it is weird, but it doesn't seem to be stopping any later optimisations